packages/react-sdk/src/components/CallControls/ToggleAudioOutputButton.tsx

```diff
@@ -11,12 +16,20 @@ export type ToggleAudioOutputButtonProps = {
 export const ToggleAudioOutputButton = (
   props: ToggleAudioOutputButtonProps,
 ) => {
+  const call = useCall();
+  const masterAudioOutputLevel = useMasterAudioOutputLevel();
+  const localParticipant = useLocalParticipant();
   const { t } = useI18n();
   const { caption = t('Speakers'), Menu = DeviceSelectorAudioOutput } = props;
+  const enabled =
+    (localParticipant?.audioOutputLevel ?? masterAudioOutputLevel) > 0;
```

Comment:
I think `audiouOutputLevel` would always stay `undefined` for the local participant. We may not need the `useLocalParticipant()` hook here.

Reply:
This one I missed

Reply:
I have adjusted the component to toggle only the master level.